### PR TITLE
cream, cilk and, soymilk now work to heal skeletons better

### PR DESCRIPTION
**Pull request by boodaliboo**
these reagents arent subtypes of milk so they didnt apply to skeletons calcium healing

:cl:  
rscadd: Added more milky reagents to calcium healing
/:cl:

### CODE REVIEW

**Comment by Cartlord** [comment]
make cilk fucking kill them

**Comment by boodaliboo** [comment]
> make cilk fucking kill them

i can do that if others think thats a good idea

**Comment by JamieD1** [comment]
Remove Cilk and Bilk and any fucking derivative of milk for snowflakes. I'll give you £5.

**Comment by boodaliboo** [comment]
> Remove Cilk and Bilk and any fucking derivative of milk for snowflakes. I'll give you £5.
will you actually pay me

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 1]
```diff
@@ -299,6 +299,44 @@
 		holder.remove_reagent(/datum/reagent/consumable/capsaicin, 2)
 	..()
 
+/datum/reagent/consumable/cilk/on_mob_life(mob/living/carbon/M)
+	if(HAS_TRAIT(M, TRAIT_CALCIUM_HEALER))
+		M.heal_bodypart_damage(1.5,0, 0)
+		. = 1
+	else
+		if(M.getBruteLoss() && prob(20))
```
Comment (Bibby0110) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm:307]: move the getBruteLoss to the outside if

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 2]
```diff
@@ -299,6 +299,44 @@
 		holder.remove_reagent(/datum/reagent/consumable/capsaicin, 2)
 	..()
 
+/datum/reagent/consumable/cilk/on_mob_life(mob/living/carbon/M)
+	if(HAS_TRAIT(M, TRAIT_CALCIUM_HEALER))
+		M.heal_bodypart_damage(1.5,0, 0)
+		. = 1
```
Comment (Bibby0110) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm:305]: Use TRUE/FALSE

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 3]
```diff
@@ -299,6 +299,44 @@
 		holder.remove_reagent(/datum/reagent/consumable/capsaicin, 2)
 	..()
 
+/datum/reagent/consumable/cilk/on_mob_life(mob/living/carbon/M)
+	if(HAS_TRAIT(M, TRAIT_CALCIUM_HEALER))
+		M.heal_bodypart_damage(1.5,0, 0)
```
Comment (Bibby0110) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]:
```suggestion
		M.heal_bodypart_damage(1.5)
```

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 4]
```diff
@@ -299,6 +299,44 @@
...
+		. = 1
+	else
+		if(M.getBruteLoss() && prob(20))
+			M.heal_bodypart_damage(1,0, 0)
+			. = 1
+	if(holder.has_reagent(/datum/reagent/consumable/capsaicin))
+		holder.remove_reagent(/datum/reagent/consumable/capsaicin, 2)
+	..()
```
Comment (Bibby0110) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm:312]: return ..()  || .

**Review by Bibby0110 — CHANGES_REQUESTED** [review]
do it for all 3

**Comment by JamieD1** [comment]
> > Remove Cilk and Bilk and any fucking derivative of milk for snowflakes. I'll give you £5.
> > will you actually pay me

Yes.

**Comment by VaelophisNyx** [comment]
beer milk should stick around since its the closest we can get to an actual mix drink used for frat-party hazing, which we **cant** use the name of

**Comment by Bibby0110** [comment]
Stale, reopen when changes completed.

**Pull request closed without merging**